spec/features/applications_spec.rb

```diff
@@ -41,14 +45,71 @@
   end
 
   scenario 'Delete' do
-    app = create(:application)
     user = create(:user)
+    app = create(:application, user: user)
     login_as(user)
 
     visit users_application_path(app)
     click_on 'Delete'
 
     expect(page).to have_content('Success')
-  end 
+  end
+
+  scenario 'non-owner attempts to view' do
+    user = create(:user)
+    app = create(:application)
+    login_as(user)
+
+    visit users_application_path(app)
+
+    expect(page.status_code).to eq(401)
+  end
+
+  scenario 'admin attempts to view' do
+    admin_user = create(:user, admin: true)
+    app = create(:application)
+    login_as(admin_user)
+
+    visit users_application_path(app)
+
+    expect(page.status_code).to eq(200)
+  end
 end
-    
+
+feature 'Admin User Approval' do
+  scenario 'new application' do
+    ActionMailer::Base.deliveries.clear
```

Comment:
create a [helper](https://github.com/18F/identity-idp/blob/master/spec/support/features/mailer_helper.rb) for clearing deliveries.